ingenialink/enums/servo.py

```diff
@@ -22,31 +22,6 @@ class SERVO_STATE(Enum):
     """Fault."""
 
 
-class SERVO_FLAGS(Enum):
```

Comment:
Didn't we remove this in another PR?

Reply:
Yes, but it was declined (the ruff config changed).  
https://github.com/ingeniamc/ingenialink-python/pull/510/